test/config/training_service.yml

```diff
@@ -78,6 +78,8 @@ paiYarn:
 pai:
   nniManagerIp:
   maxExecDuration: 15m
+  maxTrialNum: 1
```

Comment:
reduced to only 1?

Reply:
Please add comment here why change maxTrialNum.

Reply:
> reduced to only 1?

1 trial job per example is enough for testing the example case in PAI. This mastrialNum=1 only works for PAI, and other features like NNICTL, assessor etc. could be tested by other pipelines.

Reply:
> Please add comment here why change maxTrialNum.

updated.